src/include/txn.i

```diff
@@ -776,8 +776,9 @@ __wt_txn_read_upd_list(WT_SESSION_IMPL *session, WT_UPDATE *upd, WT_UPDATE **upd
             continue;
         upd_visible = __wt_txn_upd_visible_type(session, upd);
         if (upd_visible == WT_VISIBLE_TRUE) {
-            if (upd->type != WT_UPDATE_BIRTHMARK)
-                *updp = upd;
+            if (upd->type == WT_UPDATE_TOMBSTONE && WT_IS_HS(S2BT(session)))
+                continue;
+            *updp = upd;
```

Comment:
I can see why you might want to ignore tombstones in history, but that seems orthogonal to the rollback-to-stable changes.  Should this have it's own ticket / commit?

Reply:
Moved these changes into a separate ticket WT-5513.